process/templates/SIG_NAME_incubating_stage.md

```diff
@@ -1,5 +1,32 @@
 ## Special Interest Group (SIG) incubation
 
 The governing body must agree that the SIG has made substantial progress on a deliverable.
-
    * Link to relevant documentation.
+
+### SIG has met all Sandbox requirement
+  * "link to sandbox PR"
+
+### List of SIG maintainers
+The project must have a minimum of three maintainers with a minimum of two different organizational affiliations.
+  * "name, affiliation, GitHub ID"
+
+### Governance
+SIG has defined group governance
+  * "link to charter or other document describe how group is managed"
+SIG must have met publicly at least 5 times
+  * "link to meeting agenda"
+SIG should have defined Contributor Guide
+  * "link to contributor guide"
```

Comment:
```suggestion
SIG has made substantial progress on deliverable
  * "link to deliverable in progress"
```

- We don't want to require SIGs to have synchronous meetings (especially not incubating SIGs)
- The contributor guide is the Gives & Gets is for Projects (I'm not sure what a SIG contributor guide would look like?)
- The suggested progress on a deliverable comes from the SIG lifecycle doc

Reply:
Since SIG synchronous meetings are optional [once in sandbox stage](https://github.com/ossf/tac/blob/main/process/sig-lifecycle.md#once-sandbox), I do wonder if we would want to optionally add a requirement along the lines of "If regular meetings were held, provide a link to the meeting agenda"?